### DISCUSSION TITLE
Community meeting time

### DISCUSSION THREAD

**Discussion by haircommander** [cri-o/cri-o#7111] [General]
Lately, the [community meeting](https://github.com/cri-o/cri-o/wiki/CRI-O-Weekly-Meeting) has been attended more sparsely than in the past, and there are some folks that used to join that haven't. I wanted to put it out there: is there a better time or frequency for folks who wish to join but can't due to conflicts/time zone issues? We won't be able to have a perfect time, but I'd love to find a time that works for more folks if possible

**Comment by haircommander** [comment]
cc @cri-o/cri-o-maintainers @kad @littlejawa @hasan4791

**Reply by kad** [reply]
For me, current schedule works ok. Once in two weeks, and it is Thursday, but quite early in the evening, so less conflicts with usually popular by west coasts time slots... Low attendance lately can be explained by summer: many people are on vacations right now. Some people taking parts of June. In Finland July is practically "dead" month... Some countries like Germany might be out in the beginning of August...

**Reply by hasan4791** [reply]
Current schedule is perfect for me as well.

**Reply by littlejawa** [reply]
Same here - the timeslot is fine. I do have a conflicting meeting (which is why sometimes I miss the cri-o meeting), but I know it's going to be hard to find a timeslot that fits everybody's planning, so I'm fine keeping this time.
